thonny/code.py

```diff
@@ -560,6 +560,19 @@ def _init_commands(self):
             include_in_toolbar=True,
         )
 
+        get_workbench().add_command(
+            "save_all_files",
+            "file",
+            _("Save All files"),
+            self._cmd_save_all_files,
+            caption=_("Save All files"),
+            default_sequence=select_sequence("<Control-q>", "<Command-q>"),
```

Comment:
Is this correct?  Control-q is the default on Windows for closing a program - not for saving all files.

Reply:
Exactly. I forgot about that. See #1053